docs/server/push_duty/tag-services.md

```diff
@@ -0,0 +1,80 @@
+# Tag Services
+
+## Manually tagging a service
+
+`addons-server` is automatically tagged when a release is published. See [release-docs](./release-docs.md) for more information.
+
+For now, `addons-frontend` is still manually tagged.
+To create and push a new tag follow these steps:
+
+Tags are of the format: ``YYYY.MM.DD``,
+
+```{note}
+The date is the date of the push, not the date of tagging.
+```
+
+## Checking out the default branch
+
+To create a new tag, you have to checkout a branch that will be used for the tag.
+This is almost always the default branch of the repository.
+
+You can find out the default branch by running
+
+```bash
+gh repo view --json defaultBranchRef --jq '.defaultBranchRef.name'
+```
+
+```{note}
+You need to have the GitHub CLI installed to run this command.
+```
```

Comment:
Or just looking at the repo on GitHub.com? It's right there as the default branch when you load the page.

Reply:
Added a link to the UI as well.